src/main/java/frc/robot/subsystems/arm/ArmConstants.java

```diff
@@ -34,8 +39,32 @@ public class ArmConstants {
     public static final double TICKS_PER_RADIAN_SHOULDER = SHOULDER_FALCON_TICKS_PER_REVOLUTION / (Math.PI * 2);
     public static final double ELBOW_FALCON_TICKS_PER_REVOLUTION = 2048 * ELBOW_GEARING;
     public static final double TICKS_PER_RADIAN_ELBOW = ELBOW_FALCON_TICKS_PER_REVOLUTION / (Math.PI * 2);
-    public static final TalonFXInvertType MAIN_CLOCKWISE = TalonFXInvertType.Clockwise;
+    public static final boolean MAIN_CLOCKWISE = true;
     public static final TalonFXInvertType AUX_CLOCKWISE = TalonFXInvertType.Clockwise;
+    public static final TalonFXConfiguration shoulderMainMotorConfig = new TalonFXConfiguration();
+    public static final TalonFXConfiguration shoulderAuxMotorConfig = new TalonFXConfiguration();
+    public static final TalonFXConfiguration elbowMainMotorConfig = new TalonFXConfiguration();
+    public static final TalonFXConfiguration elbowAuxMotorConfig = new TalonFXConfiguration();
+    public static final double SHOULDER_FEED_FORWARD_MULTIPLIER = 0;
+    public static final double ELBOW_FEED_FORWARD_MULTIPLIER = 0;
+    public static final int SHOULDER_VELOCITY = 0;
+    public static final int ELBOW_VELOCITY = 0;
+
+    public static final Slot0Configs SHOULDER_PID= new Slot0Configs()
+                .withKP(ArmConstants.shoulderP)
+                .withKI(ArmConstants.shoulderI)
+                .withKD(ArmConstants.shoulderD);
+
+    public static final Slot1Configs ELBOW_PID = new Slot1Configs()
+                .withKP(ArmConstants.elbowP)
+                .withKI(ArmConstants.elbowI)
+                .withKD(ArmConstants.elbowD);
+
+    public static final double SENSOR_TO_MECHANISM_RATIO= 1;
```

Comment:
This is used to calculate gear ratio in the motor. When using the `getPosition()` function it's necessary, because it returns a position times the gear ratio. However, I believe that the parallel function used in the phoenix 5 version was `getRotorPosition()`, with the gear ratio taken into account later. Here https://pro.docs.ctr-electronics.com/en/latest/docs/migration/migration-guide/status-signals-guide.html#using-status-signals.